templates/CRM/Event/Form/EventFees.tpl

```diff
@@ -68,7 +68,7 @@
                     <td class="label">{$form.financial_type_id.label}<span class="crm-marker"> *</span></td>
                     <td>{$form.financial_type_id.html}<br /><span class="description">{ts}Select the appropriate financial type for this payment.{/ts}</span></td>
                 </tr>
-                <tr class="crm-event-eventfees-form-block-total_amount"><td class="label">{$form.total_amount.label}</td><td>{$form.total_amount.html|crmMoney:$currency}</td></tr>
+                <tr class="crm-event-eventfees-form-block-total_amount"><td class="label">{$form.total_amount.label}</td><td>{$form.total_amount.html}</td></tr>
```

Comment:
This is the warning I saw yesterday (two days ago). Again just no dollar sign.

In another locale the javascript when you choose the price formats it with en_US separators but it accepts it either way. And if you have mixed currencies the whole form gets confused.